.github/workflows/publish.yml

```diff
@@ -253,6 +253,16 @@ jobs:
         ANDROID_KEYSTORE_PASSWORD: ${{ secrets.ANDROID_KEYSTORE_PASSWORD_CHIS_NE }}
         ANDROID_KEY_PASSWORD: ${{ secrets.ANDROID_KEY_PASSWORD_CHIS_NE }}
 
+    - name: Bundle cht_rci
+      uses: maierj/fastlane-action@v1.4.0
+      with:
+        lane: bundle
+        options: '{ "flavor": "cht_rci" }'
+      env:
+        ANDROID_KEYSTORE_PATH: cht_rci.keystore
+        ANDROID_KEYSTORE_PASSWORD: ${{ secrets.ANDROID_KEYSTORE_PASSWORD_CHT_RCI }}
```

Comment:
Have these secrets been shared with the repo managers already?
Please refer to [this section](https://docs.communityhealthtoolkit.org/core/guides/android/branding/#:~:text=Use%20a%20safe%20channel%20to%20send%20the%20environment%20variables%20to%20the%20manager%20in%20charge) in the guide.

Reply:
@binokaryg ,  i haven't done it yet. Who is that manager and how to contact him?

Reply:
@binokaryg ? what about this?

Reply:
I can help you add those keys @gagbodamakou

Reply:
Hi  @derickl, Thank you for your support.
Is it possible to have [1 min cal](https://meet.google.com/xxd-gnrn-twq?authuser=0)l to understand this process? it is a little bit unclear for me.
thks

Reply:
I can join on the link in the next 45 min. Is that OK @gagbodamakou ?

Reply:
I will be in a meeting at that  time, let say after 2pm GMT +1, if you are available

Reply:
That works

Reply:
> Hi @gagbodamakou I got caught up with other engagements. Let me know when you would be available within the hour

Great, I am now free of all meetings

Reply:
Seems we keep missing each other. I'm on that link for the next 30 min @gagbodamakou